cfn/README-GD.md

```diff
@@ -37,8 +37,12 @@ In order to verify the user has administrator permissions:
 1. Select the user in `AIMS User` section. **Note** you can start typing a name in the search box to find an appropriate user.
 1. Once found check that `user role` list under the `Edit an AIMS User` section has `Administrator` role selected.
 
-The following procedure assumes a Linux-based local machine using [curl](https://curl.haxx.se/) and 
-[jq](https://stedolan.github.io/jq/). For Windows please use command line and windows versions of [curl](https://curl.haxx.se/download.html) and [jq](https://stedolan.github.io/jq/download/).
+
+Depending on operation system, please proceed with the Unix (MacOS, Linux) or with Windows section.
```

Comment:
Maybe something like:
"Depending on the operating system, please go to respective section: Unix (MacOS, Linux) or Windows.